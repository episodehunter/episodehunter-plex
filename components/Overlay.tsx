import * as React from 'react';

export function Overlay() {
  const style = {
    zIndex: 100,
    height: '100%',
    width: '100%',
    backgroundColor: 'rgba(0, 0, 0, 0.8)',
    position: 'absolute',
    top: 0,
    left: 0
  };
  return (
    <div style={style}></div>
  );
}
