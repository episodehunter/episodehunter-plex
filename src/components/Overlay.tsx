import * as React from 'react';

const style: React.CSSProperties = {
  zIndex: 100,
  height: '100%',
  width: '100%',
  backgroundColor: 'rgba(0, 0, 0, 0.8)',
  position: 'absolute',
  top: 0,
  left: 0
};

export function Overlay() {
  return <div style={style} />;
}
