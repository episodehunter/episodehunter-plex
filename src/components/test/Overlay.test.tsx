import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { Overlay } from '../Overlay';

test('that Overlay renders', () => {
  const tree = renderer.create(
    <Overlay />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
