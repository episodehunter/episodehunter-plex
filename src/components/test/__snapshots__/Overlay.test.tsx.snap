// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`that Overlay renders 1`] = `
<div
  style={
    Object {
      "backgroundColor": "rgba(0, 0, 0, 0.8)",
      "height": "100%",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "100%",
      "zIndex": 100,
    }
  }
/>
`;
